watchpoints.c

```diff
@@ -215,8 +200,8 @@ watchpoints_read(struct file *file, char __user * user_buffer,
 		
 		while(length && *output_pointer) {
```

Comment:
There has to be a smarter way to do it than put_user-ing every single character in the string... (I don't know about it though)